dotnet/src/InternalUtilities/planning/PlannerInstrumentation.cs

```diff
@@ -109,38 +110,49 @@ internal static partial class PlannerInstrumentation
     }
 
     #region CreatePlan Logging helpers
+#pragma warning disable SYSLIB1006 // Multiple logging methods cannot use the same event id within a class
     [LoggerMessage(
         EventId = 0,
         Level = LogLevel.Information,
         Message = "Plan creation started.")]
     static partial void LogCreatePlanStarted(this ILogger logger);
 
     [LoggerMessage(
-        EventId = 1,
+        EventId = 0,
```

Comment:
Why are all of these event ids being set to zero?

Reply:
Copied from another response:
These numbers don't carry any meaning at the moment. I think we need to audit the entire SK project (all places where logs are generated) to determine the event code for each event if we want to identify the events. For now, I have change all of them to 0, which is the default if you are just using ILogger.LogXXX. And I have also suppressed the warning.

It will become very trivial to maintain and identify each log event with a unique ID. For now, I think we can use the default. Please correct me if I am wrong.

Reply:
My only question would be if changing these to non-zero in the future constitutes a breaking change? My understanding is that it wouldn't be since they are all set to zero now, no one will be writing queries utilizing specific event IDs. @TaoChenOSU can you confirm this?